lib/final-api/endpoint/post/ddtf_tests/post_ddtf_tests.rb

```diff
@@ -0,0 +1,37 @@
+require 'final-api/ext/hash'
+
+module FinalAPI::Endpoint
+  module DDTF
+    class PostDdtfTests
+      class << self
+        public
+        def get_new_build_params(test_data, tsd)
```

Comment:
rename please to `new_builds_params`. It should be connecte with ddtf_helpers file.